flake/packages.nix

```
@@ -28,6 +22,18 @@
           };
           palette-generator = pkgs.callPackage ../palette-generator { };
         }
+        # For any output attrs normally defined by the root flake configuration,
+        # any exceptions must be manually propagated from the `dev` partition.
+        #
+        # NOTE: Attrs should be explicitly propagated at the deepest level.
+        # Otherwise the partition won't be lazy, making it pointless.
+        # E.g. propagate `packages.${system}.foo` instead of `packages.${system}`
+        # See: https://github.com/hercules-ci/flake-parts/issues/258
+        (lib.optionalAttrs (partitionStack == [ ] && pkgs.stdenv.hostPlatform.isLinux) (
+          lib.genAttrs (import ../stylix/testbed/autoload.nix) (
+            name: config.partitions.dev.module.flake.packages.${system}.${name}
+          )
```

`testbed/autoload.nix` will return a set of name->module. Here we only need the names, but for efficiency should use mapAttrs:

```suggestion
          lib.mapAttrs (
            name: _: config.partitions.dev.module.flake.packages.${system}.${name}
          ) (import ../stylix/testbed/autoload.nix)
```

I'd also like to refactor things so that we have an internal flake-parts option for the autoloaded testbed modules set; this could be re-used by both partitions.

Although DRY-er it doesn't improve efficiency, because the partition will always re-evaluate the entire flake module configuration anyway. So importing directly is fine too.